instruments/tests/test_abstract_inst/test_electrometer.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Module containing tests for the abstract electrometer class
+"""
+
+# IMPORTS ####################################################################
+
+
+import pytest
+
+import instruments as ik
+from instruments.tests import expected_protocol
+
+
+# TESTS ######################################################################
+
+
+@pytest.fixture
+def em(monkeypatch):
+    """Patch and return electrometer class for direct access of metaclass."""
+    inst = ik.abstract_instruments.Electrometer
+    monkeypatch.setattr(inst, '__abstractmethods__', set())
```

Comment:
All of these tests monkeypatch the `__abstractmethods__` attribute to return an empty set. This allows for direct access to the metaclass.

Reply:
huh, interesting. :+1: